tests/integration/update_cluster/karpenter/in-v1alpha2.yaml

```diff
@@ -116,4 +116,5 @@ spec:
   role: Node
   subnets:
   - us-test-1a
-
+  kubelet:
```

Comment:
This is just for testing this propagates to the template?

Reply:
correct, I've removed it since we'll no longer recognize these fields